astropy/units/typing_utils.py

```diff
@@ -26,10 +30,18 @@
 
 
 def generate_stub(output_dir: Path, module: ModuleType) -> None:
-    members = {name: type(getattr(module, name)) for name in module.__all__}
+    members = {}
+    functions = {}
+    for name in module.__all__:
+        cls = type(getattr(module, name))
+        if cls.__name__ == "function":
+            # Neither of the functions in unit definition modules have any parameters.
```

Comment:
Will the stubs checker fail if there ever would be functions with parameters? If so, then this is fine, otherwise we need to check and raise.

Reply:
Actually, in a case like this I'm also fine with simply `assert`ing that there are no parameters.

Reply:
Why would I want to add an `assert` here? I don't want problems with generating the stubs to cause installation failures.